### PR TITLE
Added link to documentation of each version

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/release-10-10-0/overview-10-10-0.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ layout: bundle
 
 These release notes for **Cumulocity IoT release 10.10.0**, include all relevant changes (migration notes, improvements and fixes) that have been implemented in Cumulocity IoT Core, Cumulocity IoT Streaming Analytics, Cumulocity IoT Machine Learning, and Cumulocity IoT DataHub since the previous GA release.
 
+You can find the corresponding product documentation for release 10.10.0 at [cumulocity.com/guides/10.10.0/welcome/intro-documentation](https://cumulocity.com/guides/10.10.0/welcome/intro-documentation/).
```
Comment (BeateRixen) [inline content/release-10-10-0/overview-10-10-0.md:10]: I would actually add https:// to the link her and in all other cases.

File: content/release-10-4-6/overview-10-4-6.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ layout: bundle
 
 These release notes for **Cumulocity IoT release 10.4.6**, include all relevant changes (improvements and fixes) that have been implemented in Cumulocity IoT Core and integrated products since the previous GA release.
 
+You can find the corresponding product documentation for release 10.4.6 at [cumulocity.com/guides/10.4.6/about-doc/intro-documentation](https://cumulocity.com/guides/10.4.6/about-doc/intro-documentation/).
```
Comment (BeateRixen) [inline content/release-10-4-6/overview-10-4-6.md:10]: I would actually not go further back then 10.7.0. We should not point people to the fact that all old documentation is still in the cloud :-).

File: content/release-10-10-0/overview-10-10-0.md (hunk 1 above)
Comment (dianaolives) [reply content/release-10-10-0/overview-10-10-0.md:10]: Done

File: content/release-10-4-6/overview-10-4-6.md (hunk 2 above)
Comment (dianaolives) [reply content/release-10-4-6/overview-10-4-6.md]: Done

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**